Server/Plugins/APIDump/Hooks/OnPlayerCrouched.lua

```diff
@@ -0,0 +1,18 @@
+return
+{
+	HOOK_PLAYER_CROUCHED =
+	{
+		CalledWhen = "Player crouched in the tick being currently processed.",
+		DefaultFnName = "OnPlayerCrouched",  -- also used as pagename
+		Desc = [[
+			This function is called in each server tick for each {{cPlayer|player}} that has crouched.
+		]],
+		Params =
+		{
+			{ Name = "Player", Type = "{{cPlayer}}", Notes = "The player who has crouched." },
+		},
+		Returns = [[
```

Comment:
The broadcast of positions is not dependent on the return value of the callbacks any more.

Reply:
My head is about to collapse, just like the Internet lolz. Mixed up too many files. Sorry. When I get the Travis CI Pass I'll correct it o7